### PR TITLE
Add SSH ingress rule for DSR scanner connectivity

### PR DESCRIPTION
**Pull request by marcransome**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

File: groups/ais/instance.tf [hunk 1]
```diff
@@ -27,6 +27,15 @@ resource "aws_vpc_security_group_ingress_rule" "admin_ingress" {
   ip_protocol       = "tcp"
 }
 
+resource "aws_vpc_security_group_ingress_rule" "ingress_dsr_scanning_ssh" {
+  security_group_id = aws_security_group.common.id
+  description       = "Allow inbound SSH connectivity for DSR scanning systems"
+  cidr_ipv4         = "172.19.12.0/22"
```
Comment (LJMRBUMP) [inline groups/ais/instance.tf:33]: Would This need to be a local variable for consistency?

Comment (LJMRBUMP) [inline groups/ais/instance.tf:33]: Would the cidr_ipv4 (line 33) be better to be a local variable for consistency?

Comment (marcransome) [reply groups/ais/instance.tf:33]: It's consistent with [similar rules in other stacks](https://github.com/companieshouse/dps-stack/blob/c9b42a68a7d259a503bf6046dc886f176e17d003/groups/dps/instance.tf#L83-L90) and given there's enough context here in the rule description (and no need for reuse of the actual value) a local variable would be largely redundant.

**Review by LJMRBUMP — APPROVED** [review]

**Pull request merged**